prow/istio-pilot-e2e.sh

```diff
@@ -29,14 +29,27 @@ set -u
 # Print commands
 set -x
 
-source ${ROOT}/prow/lib.sh
-setup_and_export_git_sha
-
-export NUM_NODES=4
```

Comment:
Is this not needed? How do we get 4 nodes for this test?

Reply:
the pool already created cluster with 4 nodes. https://github.com/sebastienvas/test-infra/blob/eba53851a4e94bda1abb2b1e50e5ba0457a750d7/boskos/configs.yaml#L12